Driverless AI/build-your-own-custom-recipe-tutorial/build-your-own-custom-recipe-tutorial.md

```diff
@@ -784,38 +880,41 @@ To get the predictions, the `predict()` function is called. The only thing neede
 
 Since the model needs to handle both classification and regression scenarios, the if-then statement in the code is used to identify each case. For classification, when the number of classes is 1 then the `model.predict(x)` gets called. Then for regression, the function `model.predict_proba(x)` gets called.
 
-6\. Copy and paste the code above for **get predictions**, below the `#Set model parameters` section of code of your .py file. Your text editor should look similar to the page below:
-
-![model-set-predictions](assets/model-set-predictions.jpg)
-
 **Things to note:**
 
 If the model you are working with is a model that can’t handle missing values, then that needs to be accounted for on your code. Suggestions include printing an error message, removing rows with missing values, or imputing missing values(replace missing values with the median or mode). **Driverless AI only accepts custom models that can handle missing values**. 
 
 Here is a sample code for handling the missing values in the prediction section:
 
-```python
+~~~python
         X = dt.Frame(X)
         for col in X.names:
             XX = X[:, col]
             XX.replace(None, self.min[col])
             X[:, col] = XX
         model, _, _, _ = self.get_model_properties()
         X = X.to_numpy()
-```
+~~~
+
+Your text editor should look similar to the page below:
+
+![extra-trees-model-1](assets/extra-trees-model-1.jpg)
```

Comment:
@james94 I think this code should be copyable!

Reply:
@5675sp yeah I agree!